src/app/views.py

```diff
@@ -108,6 +109,11 @@ def submitNewLicense(request):
                     licenseName = form.cleaned_data['fullname']
                     licenseIdentifier = form.cleaned_data['shortIdentifier']
                     licenseOsi = form.cleaned_data['osiApproved']
+                    isException = form.cleaned_data['isException']
+                    if isException == "False":
```

Comment:
What if the value given is `false`? Use it like 
`isException.lower() == "true" then true else false` instead